.github/workflows/labeler.yml

```diff
@@ -31,20 +30,20 @@ jobs:
     name: Label pull requests
     runs-on: ubuntu-latest
     steps:
-    # In order to get back the negated matches like in the old config,
```

Comment:
can we revert these indentation changes to make it easier to backport?

Reply:
I have reverted for this one. Thanks